crates/subspace-farmer/src/commitments/commitment_databases.rs

```diff
@@ -40,10 +37,14 @@ impl Deref for DbEntry {
     }
 }
 
-#[derive(Debug)]
 pub(super) struct CommitmentDatabases {
     base_directory: PathBuf,
-    databases: LruCache<Salt, Arc<DbEntry>>,
+    first_db: RwLock<Option<Arc<DbEntry>>>,
```

Comment:
This is the shortest and simplest I could write them -> `RwLock<Option<Arc<DbEntry>>>` but seems a mouthful. If there is a way to shorten this and make it work, I would be glad.

Reply:
Also, in `DbEntry`, we are using `Option`. 
So basically, we have `RwLock<Option<Arc<Mutex<Option<Arc<Db>>>>>` 
LOL. I think this should be simplified. 
The reason for 2 options is I guess the following:
- We don't know if there is a `first` or `second` database present. Also, if they are present, it could be that their inner `db` could be empty as well, because we are replacing them from time to time. So maybe, there is no shorter expression than we currently have.